code/game/objects/items/weapons/tools/tape.dm

```diff
@@ -35,6 +35,15 @@
 	spawn_frequency = 8
 	rarity_value = 24
 	spawn_tags = SPAWN_TAG_TOOL_ADVANCED
+	
+/obj/item/weapon/tool/tape_roll/glue
+	name = "superglue"
+	desc = "A bucket of milky white fluid. Can be used to stick things together, but unlike tape, it cannot be used to seal things."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "glue"
+	tool_qualities = list(QUALITY_ADHESIVE = 40, QUALITY_CAUTERIZING = 5) // Better than duct tape, but can't seal things and is mostly used in crafting - also, it's clue, so it can be used as an extremely shitty way of sealing wounds 
```

Comment:
```suggestion
	tool_qualities = list(QUALITY_ADHESIVE = 40, QUALITY_CAUTERIZING = 5) // Better than duct tape, but can't seal things and is mostly used in crafting - also, it's glue, so it can be used as an extremely shitty way of sealing wounds 
```

Reply:
pfghhh